app/routes/events/utils.js

```diff
@@ -1,36 +1,45 @@
 // @flow
 import { pick } from 'lodash';
 import moment from 'moment-timezone';
-import type { TransformEvent, EventType } from 'app/models';
+import type { TransformEvent } from 'app/models';
 
-export const eventTypes = {
-  company_presentation: 'Bedriftspresentasjon',
+// Value type based on the EVENT_CONSTANTS
+export type eventTypes = $Values<typeof EVENT_CONSTANTS>;
+
+// Current eventTypes
+export const EVENT_CONSTANTS: eventTypes = {
+  company_presentation: 'Bedpres',
   lunch_presentation: 'Lunsjpresentasjon',
+  alternative_presentation: 'Alternativt bedpres',
   course: 'Kurs',
-  kid_event: 'KID-arrangement',
   party: 'Fest',
   social: 'Sosialt',
-  other: 'Annet',
-  event: 'Arrangement'
+  event: 'Arrangement',
+  kid_event: 'KID-arrangement',
+  other: 'Annet'
 };
 
-export const EVENT_TYPE_TO_STRING = (eventType: EventType) => {
-  return eventTypes[eventType] || eventTypes['other'];
+// Returns the sting representation of an eventType
```

Comment:
I hate when the eventType stings :trollface:

Reply:
🙃 `:set spell spelllang=en_us` 🙃